cpp/oneapi/dal/algo/kmeans/BUILD

```diff
@@ -17,17 +17,33 @@ dal_module(
 
```

Comment:
It's not related to PCA anyhow. Please check branches.

Reply:
I've added support for DPC++ tests in Bazel to be able to run DPC++ PCA tests and had to fix existing rules/tests.